packages/bvaughn-architecture-demo/components/sources/AutoComplete/AutoCompleteList.tsx

```diff
@@ -1,11 +1,13 @@
-import { createPauseResult as Pause, Property, Scope } from "@replayio/protocol";
+import { PauseId, Property } from "@replayio/protocol";
 import { RefObject, useContext, useEffect, useMemo, useRef, useState } from "react";
 import { FixedSizeList as List } from "react-window";
 
 import Popup from "bvaughn-architecture-demo/components/Popup";
-import useCurrentPause from "bvaughn-architecture-demo/src/hooks/useCurrentPause";
+import useCurrentPauseIdSuspense from "bvaughn-architecture-demo/src/hooks/useCurrentPause";
```

Comment:
Seems like we should rename `bvaughn-architecture-demo/src/hooks/useCurrentPause` (and the function it exports) rather than just renaming the imports? 

As it is, it would be pretty easy to add new imports _without_ the "suspense" suffix.